backend/clubs/models.py

```diff
@@ -1895,6 +1896,42 @@ def send_confirmation_email(self):
             )
 
 
+class TicketTransfer(models.Model):
+    """
+    Represents a transfer of ticket ownership, used for bookkeeping
+    """
+
+    ticket = models.ForeignKey(
+        Ticket, related_name="transfers", on_delete=models.CASCADE
+    )
+    sender = models.ForeignKey(
+        get_user_model(), related_name="sent_transfers", on_delete=models.CASCADE
+    )
+    receiver = models.ForeignKey(
+        get_user_model(), related_name="received_transfers", on_delete=models.CASCADE
+    )
```

Comment:
Are we sure `CASCADE` is the intended behavior on deletion? If a ticket's been transferred, it's been bought & exchanged. Allow for its deletion seems a little unsafe. Something like `RESTRICT` or `PROTECT` might work better. 

I guess the same would go for transaction records. Let me know what you think.

Reply:
Agree CASCADE is too strong,..transaction records set null on delete

Reply:
Was actually talking to @rm03 about this -- what if we set it to `PROTECT` for transaction and exchange records? This would disallow ticket deletion without first deleting the associated receipts. 

Existence of a `TicketTransactionRecord` or `TicketTransfer` implies that the ticket has been bought & is in circulation. As long as that is true, IMO the referenced ticket shouldn't be deletable.

Reply:
Didn't know this existed: yes, even better. Nice find.

What is the behaviour if someone tries to delete a ticket that's protected by `PROTECT`? An error?

Reply:
Reason I used `CASCADE` is that a transfer record is essentially useless if we are missing one of the users or the ticket. Wasn't aware `PROTECT` existed, definitely the best choice here

Reply:
Makes sense. I think we can get away with setting it to `NULL` for the recipient or the sender, but the ticket reference should be `PROTECT`.